### PR TITLE
Feature/update readme

### PR DESCRIPTION
**Pull request by sunnyqianzhang**
Update README.md for better clarification.
@cgx027 @RackHD/corecommitters @nortonluo @iceiilin @pengz1

### CODE REVIEW

File: docs/rackhd/api-usage-examples.rst [hunk 1]
```diff
@@ -21,21 +21,26 @@ This example provides instructions on how to flash a BMC image on a Quanta (node
 2. Post the obm settings if they don't already exist for the node. An example on how to do this is shown in Section 7.1.8.1 here http://rackhd.readthedocs.io/en/latest/tutorials/vagrant.html#adding-a-sku-definition Section 7.1.8.1
 
 
-3. The firmware files need to be built into a SKU package
+3. Quanta-T41, Quanta-D51 Firmware upgrades:
```
Comment (anhou) [inline docs/rackhd/api-usage-examples.rst]: maybe the information of which platforms support firmware upgrade could be locate at the beginning of this section

File: docs/rackhd/api-usage-examples.rst [hunk 2]
```diff
@@ -21,21 +21,26 @@ This example provides instructions on how to flash a BMC image on a Quanta (node
...
+   Firmware upgrade process for these platforms is contained in the README.md file in their folders under on-skupack repositories.
+   Dell R730/R730XD, R630, C6320 Firmware upgrade(To be updated)
+
+4.  The firmware files need to be built into a SKU package
   
      **Build SKU Package**
      ::
      
-         $ ./build-package.bash <sku_pack_directory> <subname>
+         $ ./build-package.bash <sku_pack_name> <subname>
```
Comment (anhou) [inline docs/rackhd/api-usage-examples.rst]: https://github.com/RackHD/on-skupack 's readme has the same 'sku_pack_directory' parameter description. is it worth to update there?

File: docs/rackhd/api-usage-examples.rst [hunk 3]
```diff
@@ -21,21 +21,26 @@ This example provides instructions on how to flash a BMC image on a Quanta (node
...
+    <sku_pack_name> must be exactly the same with one the platforms listed on the root folder of on-skupack. E.g. it can be quanta-d51-1u, quanta-t41,dell-r630, etc. <subname>
+    can be any name you like. The new tarball is built in a folder called tarballs within the same directory
     
      ::
         
         $ /tarballs 
           sku_pack_directory_subname.tar.gz
           
-4. The SKU package that was built needs to be registered            
+5. The SKU package that was built needs to be registed
```
Comment (sunnyqianzhang) [inline docs/rackhd/api-usage-examples.rst]: Thanks Andrew for all the comments. I made a mistake and put the api-usage-example.rst to this PR, and this code change you commented  should be in this PR https://github.com/RackHD/docs/pull/354. The typo has been fixed.

File: docs/rackhd/api-usage-examples.rst (hunk 2 above)
Comment (sunnyqianzhang) [inline docs/rackhd/api-usage-examples.rst]: I made the same name changes in README.md of on-sku. The reason is that as a user, when I first use the command, I was confused by the term directory, then I referred to the test plan from aegies team and their US peer, and found <sku_pack_name> easier to understand, so I decided to make the change. But I do agree that as a variable name, this change doesn't matter much.

File: docs/rackhd/api-usage-examples.rst (hunk 1 above)
Comment (sunnyqianzhang) [inline docs/rackhd/api-usage-examples.rst]: Agree, will made this change when Dell firmware upgrade is supported.

**Comment by cgx027** [comment]
:+1:  looks good to me.

File: README.md [hunk 4]
```diff
@@ -16,9 +16,8 @@ build the docs
     cd docs
     make html
 
-to auto-rebuild docs while you're editing
+to auto-rebuild docs while you're editing, also at the same dirctory (directory named docs), execute
```
Comment (yyscamper) [inline README.md]: typo: dirctory

**Comment by anhou** [comment]
:+1:  with Felix's comments fixed

**Pull request merged**